test/y_signed_transfer_manager.js

```diff
@@ -0,0 +1,323 @@
+        //         account_investor2,
+        //         web3.utils.toWei("1", "ether"),
+        //         //account_investor1
+        //         token_owner_pk
+        //     );
+
+        //     await catchRevert(I_SignedTransferManager.invalidSignature(account_investor1, account_investor2, web3.utils.toWei("1", "ether"), sig, {from: account_investor1}));
+        // });
+
+        // it("should not be able to invalid siganture if not the original signer even if it's in the signer list", async () => {
+
+        //     const sig = signDataVerifyTransfer(
+        //         I_SignedTransferManager.address,
+        //         account_investor1,
+        //         account_investor2,
+        //         web3.utils.toWei("1", "ether"),
+        //         account_investor4
+        //     );
+
+        //     await catchRevert(I_SignedTransferManager.invalidSignature(account_investor1, account_investor2, web3.utils.toWei("1", "ether"), sig, {from: account_investor3}));
+        // });
+
+        it("should be able to invalid siganture if sender is the signer and is in the signer list", async () => {
+            const sig = signDataVerifyTransfer(
+                I_SignedTransferManager.address,
+                account_investor1,
+                account_investor2,
+                web3.utils.toWei("1", "ether"),
+                token_owner
+                // token_owner_pk
+            );
+
+            console.log("token owner is "+ token_owner);
+
+            await I_SignedTransferManager.invalidSignature(account_investor1, account_investor2, web3.utils.toWei("1", "ether"), sig, {from: token_owner});
+            console.log("2");
+            assert.equal(await I_SignedTransferManager.checkSignatureIsInvalid(sig), true);
+        });
+
+        it("should not allow transfer if the sig is already used", async () => {
+           const sig = signDataVerifyTransfer(
+                I_SignedTransferManager.address,
+                account_investor1,
+                account_investor2,
+                web3.utils.toWei("1", "ether"),
+                token_owner
+            );
+            // I_SignedTransferManager.verifyTransfer(account_investor1, account_investor2, web3.utils.toWei("1", "ether"), sig, true, {from: token_owner});
+            await catchRevert(I_SignedTransferManager.verifyTransfer(account_investor1, account_investor2, web3.utils.toWei("1", "ether"), sig, true, {from: token_owner}));
+        });
+
+        it("should allow transfer with valid sig", async () => {
+            const sig = signDataVerifyTransfer(
+                I_SignedTransferManager.address,
+                account_investor1,
+                account_investor2,
+                web3.utils.toWei("1", "ether"),
+                account_investor3
+            );
+           
+            await I_SignedTransferManager.verifyTransfer(account_investor1, account_investor2, web3.utils.toWei("1", "ether"), sig, false, {from: account_investor3});
```

Comment:
Don't directly call `I_SignedTransferManager.verifyTransfer`. Add Signed Transfer Manager to the Security token and try doing actual transfers.
Keep a couple direct calls with _isTransfer = true as well to test that functionality.